adaptive_breakpoints/example/lib/main.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:adaptive_breakpoints/adaptive_breakpoints.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        body: Column(
+          children: [
+            AdaptiveContainer(
+              windowLimit: AdaptiveWindowType.xs,
+              child: Text('This is a extra small window'),
```

Comment:
Same feedback as in the readme regarding the text in the example

Reply:
Done